### DISCUSSION TITLE
I would be very interested in support for Microsoft OneDrive.

### DISCUSSION THREAD

**Discussion by zimu96** [IceWhaleTech/CasaOS#1359] [Ideas]
"I want to stress that I would be very interested in support for Microsoft OneDrive. With 1TB of space available it would be great to be able to sync everything easily."

**Comment by thedroidkid** [comment]
Just so you know you can connect your OneDrive in the Files app

**Reply by baltazartroisville** [reply]
could someone tell me if it is possible to use onedrive as a docker volume for other apps after adding it with the files app? 😸
